perma_web/perma/celery_tasks.py

```diff
@@ -1546,6 +1546,22 @@ def deactivate_expired_sponsored_users(expiration_date=None):
         logger.info(f"Found no sponsorships that expired on {expiration_date}.")
 
 
+@shared_task
+def remove_expired_organization_user_affiliations(expiration_date=None):
+    """
+    Removes affiliation of organization users whose affiliation expired
+    """
+    if not expiration_date:
+        expiration_date = timezone.now().date()
+
+    deleted_affiliations = UserOrganizationAffiliation.objects.filter(expires_at__lt=expiration_date).delete()
+
+    if deleted_affiliations[0] > 0:
```

Comment:
Should we add another condition here to avoid erroring out when `deleted_affiliations` is empty?
```suggestion
    if deleted_affiliations and deleted_affiliations[0] > 0:
```

Reply:
Great suggestion on the label naming. I made the change for this and also for a similar label I had created for the registrar sponsorship form. I also made the change to both to use logger.exception().

When it comes to deleting affiliations, if no db objects are impacted, meaning nothing is deleted, then `deleted_affiliations[0]` will be `0`. delete() returns a tuple with its first item being the number of objects deleted.